src/tabs/__integ__/tabs.test.ts

```diff
@@ -37,7 +37,7 @@ class TabsPage extends BasePageObject {
   }
 
   paginationButton(direction: string, scrollable = false): string {
-    return `.${styles[`pagination-button-${direction}${scrollable ? '-scrollable' : ''}`]}`;
+    return `form .${styles[`pagination-button-${direction}${scrollable ? '-scrollable' : ''}`]}`;
```

Comment:
after adding a forth tab to the second tabs container it also has pagination button in small screens, this addition is to constraint the pagination tests to the first tabs container

Reply:
This does not look reliable or self-explanatory. Can we annotate tabs with specific ids?

Reply:
make sense! updated